test/python_lib/test_python_bindings.py

```diff
@@ -0,0 +1,58 @@
+# (C) Copyright IBM 2023, 2024.
+#
+# This code is part of Qiskit.
+#
+# This code is licensed under the Apache License, Version 2.0 with LLVM
+# Exceptions. You may obtain a copy of this license in the LICENSE.txt
+# file in the root directory of this source tree.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Unit tests for the dialect python bindings
+"""
+import pytest  # noqa: F401
+
+from qss_compiler.mlir.ir import *  # noqa: F401, F403
+from qss_compiler.mlir.ir import (
```

Comment:
Is this import required given the `*` import above?

Reply:
I went back and forth on the imports a bit here. On the one hand it's nice to have everything available because a lot more tests need to be written (similarly in the other files more extensions/bindings need to be created (I think)). On the other hand flake complains because I didn't explicitly import everything I'm using. So I decided to compromise by having `import *` as well as the specific imports that are used. I think that will ease development somewhat. I'm fine removing the `import *` if desired.

Reply:
Done in https://github.com/openqasm/qe-compiler/pull/276/commits/9694e7ac7e3e62f16c2ce291d8ab7b80702205b1